AmahiAnywhere/AmahiAnywhere/Data/Remote/Model/Server.swift

```diff
@@ -16,4 +16,10 @@ public class Server: EVNetworkingObject {
     public var name: String? =              nil
     public var session_token: String? =     nil
     public var active: Bool =               false
+    
+    override public func setValue(_ value: Any!, forUndefinedKey key: String) {
+        if key == "active" {
```

Comment:
This is not required here. It automatically parses and sets the value for `active` key.

Reply:
At work, I have had cases where EVReflection doesn't set Bool values for some reason. In such cases, the default `false` value will be set. I have experienced this many times. It might be difficult to figure out this out later if it becomes an issue. Ensuring the value is set by overriding setValue is just a precaution that I have adopted and it has no drawback.